### PR TITLE
Feature/redfish protocol support

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

File: xml/installation-installation-ironic-ironic_provisioning.xml [hunk 1]
```diff
@@ -92,6 +97,74 @@
...
+    </listitem>
+   </itemizedlist>
+  </note>
+  <para>
+   Enable the Redfish driver with the following steps:
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Install the Sushy library on the ironic conductor nodes. Sushy is a Python
```
Comment (asettle) [inline xml/installation-installation-ironic-ironic_provisioning.xml]: Ironic conductor*

File: xml/installation-installation-ironic-ironic_provisioning.xml [hunk 2]
```diff
@@ -92,6 +97,74 @@
...
+    </para>
+    <screen>[DEFAULT]
+...
+enabled_hardware_types = ipmi,redfish
+enabled_power_interfaces = ipmitool,redfish
+enabled_management_interfaces = ipmitool,redfish</screen>
+   </step>
+   <step>
+    <para>
+     Restart the ironic conductor service:
```
Comment (asettle) [inline xml/installation-installation-ironic-ironic_provisioning.xml]: ironic-conductor*

File: xml/installation-installation-ironic-ironic_provisioning.xml [hunk 3]
```diff
@@ -154,6 +227,77 @@ Deploy_ramdisk : openstack-ironic-image.x86_64-8.0.0.iso</screen>
...
+      portion of the URL, and can optionally include the scheme. If the scheme
+      is missing, HTTPS is assumed.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>redfish_system_id (required)</term>
+    <listitem>
+     <para>
+      The canonical path to the System resource that the driver will interact
```
Comment (asettle) [inline xml/installation-installation-ironic-ironic_provisioning.xml]:
```suggestion
The canonical path to the system resource that the driver interacts
```

File: xml/installation-installation-ironic-ironic_provisioning.xml [hunk 4]
```diff
@@ -154,6 +227,77 @@ Deploy_ramdisk : openstack-ironic-image.x86_64-8.0.0.iso</screen>
...
+      with. It should include the root service, version and the unique resource
+      path to the System. For example,<literal> /redfish/v1/Systems/1</literal>.
```
Comment (asettle) [inline xml/installation-installation-ironic-ironic_provisioning.xml]: the system*

Comment (asettle) [reply xml/installation-installation-ironic-ironic_provisioning.xml]: I see why it's uppercase, just seems a bit weird to put in the sentence too. Thoughts?

File: xml/installation-installation-ironic-ironic_provisioning.xml [hunk 5]
```diff
@@ -154,6 +227,77 @@ Deploy_ramdisk : openstack-ironic-image.x86_64-8.0.0.iso</screen>
...
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>redfish_user name (recommended)</term>
```
Comment (asettle) [inline xml/installation-installation-ironic-ironic_provisioning.xml]: mark up "redfish_user"?

File: xml/installation-installation-ironic-ironic_provisioning.xml [hunk 6]
```diff
@@ -154,6 +227,77 @@ Deploy_ramdisk : openstack-ironic-image.x86_64-8.0.0.iso</screen>
...
+    <listitem>
+     <para>
+      User account with admin/server-profile access privilege.
```
Comment (asettle) [inline xml/installation-installation-ironic-ironic_provisioning.xml]: admin and server-profile

File: xml/installation-installation-ironic-ironic_provisioning.xml [hunk 7]
```diff
@@ -154,6 +227,77 @@ Deploy_ramdisk : openstack-ironic-image.x86_64-8.0.0.iso</screen>
...
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>redfish_password (recommended)</term>
```
Comment (asettle) [inline xml/installation-installation-ironic-ironic_provisioning.xml]: mark up? <literal>redfish_password</literal>

File: xml/installation-installation-ironic-ironic_provisioning.xml [hunk 8]
```diff
@@ -154,6 +227,77 @@ Deploy_ramdisk : openstack-ironic-image.x86_64-8.0.0.iso</screen>
...
+    <listitem>
+     <para>
+      User account password.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>redfish_verify_ca (optional)</term>
```
Comment (asettle) [inline xml/installation-installation-ironic-ironic_provisioning.xml]: <literal>redfish_verify_ca</literal>

File: xml/installation-installation-ironic-ironic_provisioning.xml (hunk 1 above)
Comment (csymons-suse) [reply xml/installation-installation-ironic-ironic_provisioning.xml]: this and other corrections made

**Pull request merged**